hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/domain/EntityId.java

```diff
@@ -113,11 +114,15 @@ public static EntityId of(String entityId, EntityTypeEnum type) {
 
     public static EntityId of(long entityShard, long entityRealm, long entityNum, EntityTypeEnum type) {
         if (entityNum == 0 && entityRealm == 0 && entityShard == 0) {
-            return null;
+            return EMPTY;
         }
         return new EntityId(entityShard, entityRealm, entityNum, type.getId());
     }
 
+    public static boolean isEmpty(EntityId entityId) {
```

Comment:
It checks for more than empty
```suggestion
    public static boolean isNullOrEmpty(EntityId entityId) {
```

Reply:
Null is another form of empty, is it not?

Reply:
I agree with Steven.  `StringUtils.isEmpty()` does a null check before a length check for instance.

Reply:
I'm okay with it if the team is.